src/NzbDrone.Core/Download/Clients/Tribler/TriblerDownloadClient.cs

```diff
@@ -0,0 +1,324 @@
+                item.SeedRatio = download.Ratio;
+
+                if (download.Eta.HasValue)
+                {
+                    if (download.Eta.Value >= TimeSpan.FromDays(365).TotalSeconds)
+                    {
+                        item.RemainingTime = TimeSpan.FromDays(365);
+                    }
+                    else if (download.Eta.Value < 0)
+                    {
+                        item.RemainingTime = TimeSpan.FromSeconds(0);
+                    }
+                    else
+                    {
+                        item.RemainingTime = TimeSpan.FromSeconds(download.Eta.Value);
+                    }
+                }
+
+                // TODO: the item's message should not be equal to Error
+                item.Message = download.Error;
+
+                // tribler always saves files unencrypted to disk.
+                item.IsEncrypted = false;
+
+                // state handling
+
+                // TODO: impossible states?
+                // Failed = 4,
+                // Warning = 5
+
+                // Queued = 0,
+                // Completed = 3,
+                // Downloading = 2,
+
+                // Paused is guesstimated
+                // Paused = 1,
+
+                switch (download.Status)
+                {
+                    case DownloadStatus.Hashchecking:
+                    case DownloadStatus.Waiting4HashCheck:
+                    case DownloadStatus.Circuits:
+                    case DownloadStatus.Exitnodes:
+                    case DownloadStatus.Downloading:
+                        item.Status = DownloadItemStatus.Downloading;
+                        break;
+                    case DownloadStatus.Metadata:
+                    case DownloadStatus.AllocatingDiskspace:
+                        item.Status = DownloadItemStatus.Queued;
+                        break;
+                    case DownloadStatus.Seeding:
+                    case DownloadStatus.Stopped:
+                        item.Status = DownloadItemStatus.Completed;
+                        break;
+                    case DownloadStatus.StoppedOnError:
+                        item.Status = DownloadItemStatus.Failed;
+                        break;
+                    default: // new status in API? default to downloading
+                        item.Message = "Unknown download state: " + download.Status;
+                        _logger.Info(item.Message);
+                        item.Status = DownloadItemStatus.Downloading;
+                        break;
+                }
+
+                // override status' if completed but progress is not finished
+                if (download.Status == DownloadStatus.Stopped && download.Progress < 1)
+                {
+                    item.Status = DownloadItemStatus.Paused;
+                }
+
+                // override status if error is set
```

Comment:
Not really necessary

Reply:
i'm i a bit uncertain if it's the pause part that is unnessesary or the comment after?

Reply:
I was referring to the comment.